src/pages/in/nft/_id.vue

```diff
@@ -0,0 +1,27 @@
+<script>
+import { checkIsValidISCNId, checkIsValidNFTClassId } from '~/util/nft';
+
+export default {
+  fetch({
+    error,
+    redirect,
+    params: { id },
+    query,
+  }) {
+    if (checkIsValidISCNId(id)) {
+      redirect({
+        name: 'in-like-id',
+        params: { id: 'iscn' },
+        query: { ...query, iscn_id: id }
+      });
+    } else if (checkIsValidNFTClassId(id)) {
+      redirect({
+        name: 'in-nft',
+        query: { ...query, class_id: id },
+      });
+    } else {
+      error({ statusCode: 400, message: 'INVALID_NFT_IDENTIFIER' });
```

Comment:
we can actually do [optional params in route](https://nuxtjs.org/docs/features/file-system-routing/#dynamic-routes) to merge this file with `index?